scripts/perftune.py

```diff
@@ -672,7 +691,12 @@ def __learn_irqs_one(self, iface):
         fp_irqs_re = re.compile("\-TxRx\-|\-fp\-|\-Tx\-Rx\-|mlx\d+\-\d+@")
         irqs = list(filter(lambda irq : fp_irqs_re.search(self.__irqs2procline[irq]), all_irqs))
         if irqs:
-            irqs.sort(key=self.__intel_irq_to_queue_idx)
+            driver_name = self.__get_driver_name(iface)
+            print("Sorting IRQs by driver: {}".format(driver_name))
+            if (driver_name.startswith("mlx")):
+                irqs.sort()
```

Comment:
I'm not sure that you can assume that IRQs numbers ordering is the same as corresponding HW queues order. 
I was sorting HW queues by their index for Intel NICs for exactly same reason.

Reply:
what is the source of truth for ordering?
parse queue ids and assign them to positions by their numbers?
if yes, then this is OK, I will do it like this, but if this is the same for intel, then I guess I will change parsing code to work slightly better

Reply:
The index in the name of the queue in `/proc/interrupts`